plonetheme/sunburst/skins/sunburst_templates/main_template.pt

```diff
@@ -13,7 +13,9 @@
         site_properties nocall:context/portal_properties/site_properties;
         ajax_load request/ajax_load | nothing;
         ajax_include_head request/ajax_include_head | nothing;
-        dummy python:request.RESPONSE.setHeader('X-UA-Compatible', 'IE=edge,chrome=1');"
+        dummy python:request.RESPONSE.setHeader('X-UA-Compatible', 'IE=edge');
```

Comment:
I don't know much about this header, why can we remove the ``chrome=1``?

Reply:
because [is outdated](https://en.wikipedia.org/wiki/List_of_HTTP_header_fields#Common_non-standard_response_fields) and leads to HTML validation errors.

in the future we must probably just remove it.

Reply:
I just checked and in some server we just remove this header with this Varnish snippet:

```
sub vcl_deliver {
    unset resp.http.X-UA-Compatible;
}
```

I think we can now get rid of it in Plone also; I'll do that on Plone 5 only.